code/modules/paperwork/paper_bundle.dm

```diff
@@ -227,39 +235,42 @@
 
 /obj/item/paper_bundle/update_desc()
 	. = ..()
-
-	if(amount > 1)
-		desc =  "[amount] papers clipped to each other."
+	if(amount == (photos - 1))
+		desc = "[photos] photos clipped together." // In case you clip 2 photos together and remove the paper
+		return
+	else if(((amount + 1) - photos) >= 2) // extra papers + original paper - photos
+		desc = "[(amount + 1) - photos] papers clipped to each other."
 	else
 		desc = "A single sheet of paper."
```

Comment:
```suggestion
	if(amount == (photos - 1))
		desc = "[photos] photos clipped together." // In case you clip 2 photos together and remove the paper
		return
	
	else if(((amount + 1) - photos) >= 2) // extra papers + original paper - photos
		desc = "[(amount + 1) - photos] papers clipped to each other."
	
	else
		desc = "A single sheet of paper."

```